serial_protocol.md

```diff
@@ -0,0 +1,78 @@
+#Protocol:
+During the first use, or when the chiplet changes environments a “NETWORK” call is expected to initialize the wifi parameters.
+
+Every time a serial connection is established we will expect a “INIT” call after which any subsequent calls can be made, until the connection is closed. 
+
+In the following examples we use $ to represent variables. For example $Host represents your firebase host.
+
+##Response Format Byte
+All responses will be prefixed with one of the following bytes signifying the response type.
+```
+  + If response is ok and a raw string value.
+  * If response is ok and a raw string value prefixed by count of bytes in response then new line.
```

Comment:
I think the more detail about the value we can easily provide the better. 

Like if you are reading a string from https:/.../client/value. Then someone writes a https:/.../client/value/child then your response will change from a raw string to a json tree and you should change your parsing to match. In that case it would be nice if the response type changed so the issue would be more obvious to you.